boxsdk/object/item.py

```diff
@@ -356,3 +356,70 @@ def metadata(self, scope='global', template='properties'):
             :class:`Metadata`
         """
         return Metadata(self._session, self, scope, template)
+
+    @api_call
+    def add_to_collection(self, collection):
+        """
+        Add the item to a collection.
+
+        :param collection:
+            The collection to add the item to.
+        :type collection:
+            :class:`Collection`
+        :return:
+            This item.
+        :rtype:
+            :class:`Item`
+        """
+        url = self.get_url()
+        params = {
+            'fields': 'collections'
```

Comment:
Perhaps we should also get the `etag` and `sequence_id` and pass the correct `If-Match` header, that way we don't have race conditions with the PUT?

Either that, or give a warning in the docstring that this method isn't safe from race conditions.

Same for `remove_from_collection`.